fixedwidth/src/main/java/org/apache/metamodel/fixedwidth/FixedWidthConfiguration.java

```diff
@@ -33,66 +33,74 @@
 /**
  * Configuration of metadata about a fixed width values datacontext.
  */
-public final class FixedWidthConfiguration extends BaseObject implements
-		Serializable {
+public final class FixedWidthConfiguration extends BaseObject implements Serializable {
 
-	private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = 1L;
 
-	public static final int NO_COLUMN_NAME_LINE = 0;
-	public static final int DEFAULT_COLUMN_NAME_LINE = 1;
+    public static final int NO_COLUMN_NAME_LINE = 0;
+    public static final int DEFAULT_COLUMN_NAME_LINE = 1;
 
-	private final String encoding;
-	private final int fixedValueWidth;
-	private final int[] valueWidths;
-	private final int columnNameLineNumber;
-	private final boolean failOnInconsistentLineWidth;
-	private final ColumnNamingStrategy columnNamingStrategy;
+    private final String encoding;
+    private final int fixedValueWidth;
+    private final int[] valueWidths;
+    private final int columnNameLineNumber;
+    private final boolean failOnInconsistentLineWidth;
+    private final boolean headerPresent;

```

Comment:
I am not so sure about that. I can image it is confusing (let's create a better name then) but this is something else. In EBCDIC files there might be a "binary header" with some meta-data that we want to skip before we can read actual data. And the actual data can (perhaps) contain the "normal" header with column names. I am not sure if this can really happen since there are "format" files for structure and column names definition.  

Example:
1. HDR 2016-06-09 ... // this is "headerPresent" 
[2. NAME, SURNAME, YEAR, ...]? // this is "columnNameLineNumber"
3. John, Doe, 1980, ...